bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/HttpServiceUtil.java

```diff
@@ -39,45 +39,44 @@ public static int getHttpServicePortSecure(final BundleContext bc) {
     }
 
     // Utility method that could be used for non-secure and secure port.
-    @SuppressWarnings("rawtypes")
     private static int getHttpServicePortProperty(final BundleContext bc, final String propertyName) {
         Object value;
         int port = -1;
 
         // Try to find the port by using the service property (respect service ranking).
+        ServiceReference<?>[] refs = new ServiceReference[0];
```

Comment:
As it is implementation specific, we can use `BundleContext#getServiceReferences(Class, String)` that returns `Collection<ServiceReference<SERVICE>>`